### PR TITLE
zh, en: comment out spec.from for Backup and spec.to for Restore in v1.4

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

comment out `spec.from` for Backup using BR and `spec.to` for Restore using BR.

not comment out these for Dumpling & Lightning

ref #2614

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.6 (TiDB Operator 1.6 versions)
- [ ] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by WangLe1321 — APPROVED** [review]

File: en/enable-tls-for-mysql-client.md [hunk 1]
```diff
@@ -395,8 +395,8 @@ You can generate multiple sets of client-side certificates. At least one set of
 
     - TidbInitializer
     - PD Dashboard
-    - Backup
-    - Restore
+    - Backup (when using Dumpling)
+    - Restore (when using Lightning)
```
Comment (Oreoxmt) [inline en/enable-tls-for-mysql-client.md:399]:
```suggestion
    - Restore (when using TiDB Lightning)
```

File: zh/enable-tls-for-mysql-client.md [hunk 2]
```diff
@@ -606,12 +606,6 @@ summary: 在 Kubernetes 上如何为 TiDB 集群的 MySQL 客户端开启 TLS。
             cluster: ${cluster_name}
             clusterNamespace: ${namespace}
             sendCredToTikv: true
```
Comment (Oreoxmt) [inline zh/enable-tls-for-mysql-client.md:608]: Please update https://github.com/pingcap/docs-tidb-operator/blob/0dde3d9f40bcde85178ed8ed9a82f6d1c275dae1/zh/enable-tls-for-mysql-client.md?plain=1#L394-L395

Comment (csuzhangxc) [reply zh/enable-tls-for-mysql-client.md:608]: https://github.com/pingcap/docs-tidb-operator/pull/2615/commits/9aea0fbe726a549f674632269c5fda7646c6e939

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**